backend/mora/service/handlers.py

```diff
@@ -222,13 +253,33 @@ def prepare_terminate(self, request: dict):
             },
         )
 
+    def prepare_amqp_message(self, request: dict):
+        super().prepare_amqp_message(request)
+        is_create = self.request_type == RequestType.CREATE
+        one_uuid_is_none = (
+            self.org_unit_uuid is None or self.employee_uuid is None
+        )
+        if one_uuid_is_none and not is_create:
```

Comment:
You don't actually need the `one_uuid_is_none` Boolean if you're checking explicitly below if each of them is None?